src/main/java/pl/umk/mat/git2befit/controller/workout/TrainingPlanController.java

```diff
@@ -102,8 +102,11 @@ public TrainingPlanController(TrainingPlanManufacture manufacture, TrainingPlanS
     }
 
     @GetMapping
-    public List<TrainingPlan> getAllTrainingPlansByUserId(@RequestHeader long userId) {
-        return trainingPlanService.getAllTrainingPlansByUserId(userId);
+    public List<TrainingPlan> getAllTrainingPlansByUserEmail(
+            @RequestHeader(value = "Authorization") String authorizationToken
+    ) {
+        String email = JWTService.parseEmail(authorizationToken);
+        return trainingPlanService.getAllTrainingPlansByUserEmail(email);
```

Comment:
Jak już wszędzie robimy resoponseEntity to się tego trzymajmy i go zwracajmy. I wywal tą logikę z emailem z kontrolera.